src/java.base/share/classes/java/util/Calendar.java

```diff
@@ -2642,7 +2642,7 @@ public static Set<String> getAvailableCalendarTypes() {
     private static class AvailableCalendarTypes {
         private static final Set<String> SET;
         static {
-            Set<String> set = new HashSet<>(3);
+            Set<String> set = HashSet.newHashSet(3);
             set.add("gregory");
             set.add("buddhist");
             set.add("japanese");
```

Comment:
This can be replaced with `SET = Set.of("gregory", "buddhist", "japanese");`.

Reply:
@naotoj Yes it can. I did a further clean up to it, please have a look.